.gitignore

```diff
@@ -0,0 +1,11 @@
+## IDE Specific
+.idea/
+out/s
+*.iml
+*.ipr
+*.iws
+
+.settings/
+.classpath
+.project
+s
```

Comment:
@takezoe than how is it edited? I used an IDE since there were **no** static site config files.